time_manager/time_manager.F90

```diff
@@ -1669,16 +1669,18 @@ end function get_ticks_per_second
 !     similar to this:
 !     if(err_msg /= '') call error_mesg('my_routine','additional info: '//trim(err_msg),FATAL)
 !   </OUT>
- subroutine get_date(time, year, month, day, hour, minute, second, tick, err_msg)
+ subroutine get_date(time, year, month, day, hour, minute, second, tick, err_msg, old_method)
```

Comment:
You should doxygenize the description of this subroutine since it performs something new.

Reply:
Added a brief @brief description